halo2_proofs/tests/compress_selectors.rs

```diff
@@ -494,13 +494,13 @@ fn test_key_compression() -> Result<(), halo2_proofs::plonk::Error> {
     // vk & pk keygen both WITH compression
     test_result(
         || test_mycircuit(true, true).expect("should pass"),
-        "acae50508de5ead584170dd83b139daf40e1026b6debbb78eb05d515173fc2dd",
+        "44130c6388df3d99263be8da4a280b426dc05f1f315d35d3827347761534bf08",
```

Comment:
Yes, you are right. @adria0 

The `proof` becomes different, since there is part of hashing `VerifyingKey` into `transcript` in proving/verifying process.
The use of `ColumnMid` brings different representation of `ConstraintSystem`, which is inner field of `VerifyingKey`.
Hence, the hash of `VerifyingKey` is different in `transcript`.
This `transcript` is further used for generating `Verifier`-providing values, like `beta`, `gamma`.
Overall, the `proof` becomes different from past version.